lib/api.ts

```diff
@@ -0,0 +1,36 @@
+var BASE_API = 'https://api.aircall.io/v1'
+
+var request = require('superagent')
```

Comment:
```suggestion
import request from 'superagent'

const BASE_API = 'https://api.aircall.io/v1'
```

Reply:
I tried first and add some issues with request typings but I found a good way to do it, thanks 😁